transcribe.py

```diff
@@ -57,16 +61,24 @@ def main(args):
 
 		transcript_path = os.path.join(args.output_path, os.path.basename(audio_path) + '.json')
 
+		if max(ylen) > args.skip_subtitles_longer_than:
+			print(f'Too large subtitles [{ylen}] [{audio_path}]. Skipping.')
```

Comment:
We don't use term `subtitles` anywhere else in the codebase. Please refer to `ref` instead

Reply:
fixed

Reply:
Please make the error message clearer, e.g. `Too large ref len: len(ylen) = {len(ylen)} > {args.max_ref_len) = max_ref_len`